network/discovery_test.go

```diff
@@ -112,13 +112,13 @@ func TestRoutingTable_Disconnected(t *testing.T) {
 		1: defaultConfig,
 	}
 
-	servers, createErr := createServers(3, paramsMap)
+	servers, createErr := createServers(2, paramsMap)
 	if createErr != nil {
 		t.Fatalf("Unable to create servers, %v", createErr)
 	}
 
 	t.Cleanup(func() {
-		closeTestServers(t, servers)
+		closeTestServers(t, servers[1:])
```

Comment:
Why not closing all test servers?

Reply:
Because we close Server 0 inside the test:

https://github.com/0xPolygon/polygon-edge/pull/374/files#diff-0c20fd327ddad3dc3b8cf75536c00e3d3010460d8359f35d84d749567d4e8a15R145

Double closing it would cause a panic